src/nspawn/nspawn.c

```diff
@@ -2037,32 +2037,43 @@ static int setup_pts(const char *dest) {
         return 0;
 }
 
-static int setup_dev_console(const char *dest, const char *console) {
-        _cleanup_umask_ mode_t u;
-        const char *to;
+static int setup_stdio(const char *console) {
```

Comment:
the parameter is always "/dev/console", no? hence should be good to just drop it and use the literal string in this function